qa/pull-tester/rpc-tests.sh

```diff
@@ -5,16 +5,19 @@ CURDIR=$(cd $(dirname "$0"); pwd)
 # Get BUILDDIR and REAL_BITCOIND
 . "${CURDIR}/tests-config.sh"
 
-export BITCOINCLI=${BUILDDIR}/qa/pull-tester/run-bitcoin-cli
 export BITCOIND=${REAL_BITCOIND}
 
 if [ "x${EXEEXT}" = "x.exe" ]; then
-  echo "Win tests currently disabled"
-  exit 0
+    export BITCOINCLI=${REAL_BITCOINCLI}
+    export EXEEXT=${EXEEXT}
+    echo 'Windows tests are disabled by default due to issue with Travis build'
+    echo 'To enable Widows tests, edit rpc-tests.sh and comment out "exit"'
+    exit
```

Comment:
s/To enable Widows/To enable Wi**n**dows/

Why the exit? At least we could disable the RPC test over the `travis.yml` file for the Win builds (IIRC RPC tests do not run for the current win builds)